index.js

```diff
@@ -1,12 +1,12 @@
-import postcss from 'postcss';
-import getCustomProperties from './lib/get-custom-properties';
-import transformProperties from './lib/transform-properties';
-import importCustomPropertiesFromSources from './lib/import-from';
-import exportCustomPropertiesToDestinations from './lib/export-to';
+import postcss from "postcss";
```

Comment:
Formatting changes like this pollute the PR and make it harder for me to read and unreasonable for me to merge. Be sure you’ve correctly configured your editor to follow the project guidelines and avoid unnecessary changes unrelated to the scope of your feature or fix.